### PR TITLE
Fix permission issue caused by tpm device when disable dynamic_ownership

### PR DESCRIPTION
**Pull request by yafu-1**
Signed-off-by: Yan Fu <yafu@redhat.com>

# Format of PR title < sub-system: summary >
e.g
*virsh_migrate: Fix unsupported direct socket mode issue*

# Check lists by category
## If the PR is new cases
- [ ] Description of the cases
- [ ] Links of libvirt features, libvirt bugs or case IDs
- [ ] Test results

## If the PR is bug cases
- [ ] Bug descriptions or bug links
- [ ] Test results

## If the PR is a trivial fix
It it is the fix of typos, comments or documents, the description of PR could be skipped.

### CODE REVIEW

File: libvirt/tests/src/svirt/dac_start_destroy.py [hunk 1]
```diff
@@ -91,6 +91,17 @@ def format_user_group_str(user, group):
     return label_str
 
 
+def set_swtpmlib_perms(swtpmlib_dir):
+    """
+    Set the perms of swtpmlib dir to allow other users to write in the dir
+
+    :param swtpmlib_dir: swtpmlib dir by default
+    """
+    cmd = "getfacl -R %s > /tmp/permis.facl" % swtpmlib_dir
```
Comment (dzhengfy) [inline libvirt/tests/src/svirt/dac_start_destroy.py:100]: We need to consider to clean up the new file `/tmp/permis.facl`.

File: libvirt/tests/src/svirt/dac_start_destroy.py [hunk 2]
```diff
@@ -206,12 +217,19 @@ def _create_user():
...
             if qemu_group:
                 qemu_conf.group = qemu_group
+                if not dynamic_ownership:
+                    qemu_conf.swtpm_group = qemu_group
             if dynamic_ownership:
                 qemu_conf.dynamic_ownership = 1
             else:
                 qemu_conf.dynamic_ownership = 0
+                if vmxml.devices.by_device_tag('tpm') is not None:
+                    swtpmlib_dir = "/var/lib/swtpm-localca"
```
Comment (dzhengfy) [inline libvirt/tests/src/svirt/dac_start_destroy.py:231]: Could you help create a parameter instead of hardcoding a directory?

**Review by dzhengfy — COMMENTED** [review]
Others LGTM. But please add the failure message before the fix and the test result with the fix.

**Comment by dzhengfy** [comment]
@yafu-1 Please also change the ignore file for your new words in comments to solve the CI failure.

**Comment by dzhengfy** [comment]
@yafu-1 , as you need more codes for the fix, I convert it to draft  to avoid others' reviewing.

File: libvirt/tests/src/svirt/dac_start_destroy.py (hunk 1 above)
Comment (yafu-1) [reply libvirt/tests/src/svirt/dac_start_destroy.py]: Updated. Thanks.

File: libvirt/tests/src/svirt/dac_start_destroy.py (hunk 2 above)
Comment (yafu-1) [reply libvirt/tests/src/svirt/dac_start_destroy.py]: Updated. Thanks.

**Comment by yafu-1** [comment]
Test result:
$avocado run dac_start_destroy..disable_dynamic_ownership --vt-type libvirt
No python imaging library installed. Screendump and Windows guest BSOD detection are disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
No python imaging library installed. PPM image conversion to JPEG disabled. In order to enable it, please install python-imaging or the equivalent for your distro.
JOB ID     : 8946bd8f49f6ee2a55d913046ad7b2bf5823056f
JOB LOG    : /var/lib/avocado/job-results/job-2022-09-14T02.39-8946bd8/job.log
 (01/68) type_specific.io-github-autotest-libvirt.dac_start_destroy.positive_test.qemu_conf_only.root_spec.with_qemu_conf.disable_dynamic_ownership.root_usr: STARTED
 (01/68) type_specific.io-github-autotest-libvirt.dac_start_destroy.positive_test.qemu_conf_only.root_spec.with_qemu_conf.disable_dynamic_ownership.root_usr: PASS (45.08 s)
 (02/68) type_specific.io-github-autotest-libvirt.dac_start_destroy.positive_test.qemu_conf_only.non_root.with_qemu_conf.disable_dynamic_ownership.qemu_usr: STARTED
 ...
 (68/68) type_specific.io-github-autotest-libvirt.dac_start_destroy.negative_test.qemu_conf_only.non_root.with_qemu_conf.disable_dynamic_ownership.no_qemu_usr: STARTED
 (68/68) type_specific.io-github-autotest-libvirt.dac_start_destroy.negative_test.qemu_conf_only.non_root.with_qemu_conf.disable_dynamic_ownership.no_qemu_usr: PASS (5.69 s)
RESULTS    : PASS 68 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2022-09-14T02.39-8946bd8/results.html
JOB TIME   : 731.60 s

**Review by chloerh — APPROVED** [review]

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**